integration/container/diff_test.go

```diff
@@ -12,22 +12,36 @@ import (
 )
 
 func TestDiff(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows", "cannot diff a running container on Windows")
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	cID := container.Run(ctx, t, apiClient, container.WithCmd("sh", "-c", `mkdir /foo; echo xyzzy > /foo/bar`))
+
+	expected := []containertypes.FilesystemChange{
+		{Kind: containertypes.ChangeAdd, Path: "/foo"},
+		{Kind: containertypes.ChangeAdd, Path: "/foo/bar"},
+	}
+
+	items, err := apiClient.ContainerDiff(ctx, cID)
+	assert.NilError(t, err)
+	assert.DeepEqual(t, expected, items)
+}
+
+func TestDiffStoppedContainer(t *testing.T) {
 	skip.If(t, testEnv.DaemonInfo.OSType == "windows", "FIXME")
```

Comment:
If we cannot diff a running container, does that mean we _could_ diff a stopped container on Windows (and so, we wouldn't have to skip this one?)

Reply:
I think we should be able to diff a stopped container, the tests suggests so, but I have no idea what "FIXME" means, I didn't go spelunking to see when and why this was added, I will, FIXME is quite criptic and doesn't really give any useful info

Reply:
LOL, and in case you were wondering "why all the skips, and not just gate them in files with `//go:build windows` build-tags; I think that was a combination;

- being able to run the test from a Linux machine against a Windows machine (I think this was needed when there were only internal builds of Windows that supported containers, so we didn't have them in our CI, but could connect to them)
- as a passive-aggressive way to say: look at all these skipped tests on Windows (hint hint, nudge nudge)

Reply:
> If we think this should work on Windows, perhaps we should try removing the skip and see if it does :)

I tried, it fails, or rather it gives a huge diff

Reply:
> Right, so I (perhaps incorrectly) assumed you added the new test to split the "won't work on windows" from "should work on windows" ones, so that we could skip the new test, but run the existing one.

Yeah no, I split to two different tests: one for diffing running containers and the second one for diffing on a stopped container, we didn't have a test for diff on a stopped container on linux and I had a bug in my code so wanted to make sure that doesn't happen again

Reply:
> I tried, it fails, or rather it gives a huge diff

OH! I think i saw that diff posted somewhere (or in a CI failure)

LOL, so I guess "diff" is just broken on Windows then, and nobody ever looked into it, and nobody complained 🥹